cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -1072,6 +1149,538 @@ def cancel(self, req):
+                for attempt in range(max_retries):
+                    try:
+                        cmd_str = '%s "shred -u %s 2>/dev/null || rm -f %s"' % (
+                            sshpass_cmd_header,
+                            quoted_remote_sudo_passwd_file_path,
+                            quoted_remote_sudo_passwd_file_path)
+                        r, _, e = bash.bash_roe(cmd_str)
+                        if r == 0:
+                            return
+                        logger.warning("attempt %d/%d to delete remote sudo passwd file failed: %s"
+                                       % (attempt + 1, max_retries, e))
+                    except Exception:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)